skia/config/SkUserConfig.h

```diff
@@ -141,7 +141,12 @@ SK_API void SkDebugf_FileLine(const char* file, int line, bool fatal,
     "%s:%d: failed assertion \"%s\"\n", \
     __FILE__, __LINE__, #cond); } } while (false)
 
-#if !defined(ANDROID)   // On Android, we use the skia default settings.
```

Comment:
How about adding only condition instead of duplicate code?
#if !defined(ANDROID) || defined(CASTANETS)